lib/niconico/live/api.rb

```diff
@@ -165,6 +165,16 @@ def get_player_status(id, public_key = nil)
           ticket: rtmp.at('ticket').inner_text,
         }
 
+        stream = status.at('stream')
+        result[:stream] = {
+          id:                stream.at('id').inner_text,
+          default_community: stream.at('default_community').inner_text,
+          base_time:         stream.at('base_time').inner_text,
+          open_time:         stream.at('open_time').inner_text,
+          watch_count:       stream.at('watch_count').inner_text,
+          comment_count:     stream.at('comment_count').inner_text
```

Comment:
I'm sorry, I forgot. Added to to_i.

id -> "lv....."
default_community -> "co....."

Please go if there is a funny place.